cves/kernel/CVE-2020-16166.yml

```diff
@@ -311,8 +243,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: ["time", "drivers"]
+  note: |
+    This vulnerability resulted in changes to timing and random files
```

Comment:
Your not mentions random files but for subsystems you mention drivers. I'm a little confused on where the mistake is. Could you possibly be a little more specific cause of the different naming?